### PR TITLE
feat(Highlights): Base highlights

### PR DESCRIPTION
**Pull request by ioanpocol**
Create, Edit, Delete highlights
Mark items for a highlights

### CODE REVIEW

**Comment by petrjasek** [comment]
can you put some description on a wiki what is the idea? seems like you're storing ids of items instead of having some flags on item. does it fit all requirements? eg can you get list of all highlighted items?

**Comment by ioanpocol** [comment]
Changed the code to save marked for highlights on the item

**Comment by amagdas** [comment]
:+1:

File: apps/highlights/service.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+class MarkedForHighlightsService(BaseService):
+    def create(self, docs, **kwargs):
+        service = get_resource_service('archive')
+        ids = []
+        for doc in docs:
+            item = service.find_one(req=None, guid=doc['marked_item'])
+            if not item:
+                ids.append(None)
+                continue
```
Comment (mugurrus) [inline apps/highlights/service.py]: Make a test to verify the situation when a doc was deleted.

File: features/highlights.feature [hunk 2]
```diff
@@ -0,0 +1,108 @@
+Feature: Highlights
+
+    @auth
+    Scenario: Create highlight
+        When we post to "desks"
+		"""
+		[{"name": "desk1"}]
+		"""	
+		Then we get new resource
```
Comment (mugurrus) [inline features/highlights.feature]: Given can be used here instead of post

**Comment by mugurrus** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

File: apps/highlights/service.py (hunk 1 above)
Comment (ioanpocol) [inline apps/highlights/service.py]: Added a test for a not existing item

**Pull request merged**